lib/mayaUsd/fileio/shading/shadingModeRegistry.h

```diff
@@ -178,6 +178,17 @@ class UsdMayaShadingModeRegistry : public TfWeakBase
         TfToken importDescription;
         bool    hasExporter = false;
         bool    hasImporter = false;
+
+        ConversionInfo() = default;
+        ConversionInfo(TfToken rc, TfToken nn, TfToken ed, TfToken id, bool he, bool hi)
+            : renderContext(rc)
+            , niceName(nn)
```

Comment:
We haven't used our naming convention for member names, so I used short argument names to avoid inventing new style here.